README.md

```diff
@@ -170,6 +170,37 @@ If you would like to get more than just the `id` from the record, you can use th
   def handle_info({:updated, MyApp.Posts.Comment, %{id: id, post_id: post_id}}, socket) do
 ```
 
+## Watching without a schema
+
+Since ecto supports working with tables withoun needed a schema, you may also want to create EctoWatch watchers without needing to create a schema like so:
+
+```elixir
+  # setup
+  {EctoWatch,
+   repo: MyApp.Repo,
+   pub_sub: MyApp.PubSub,
+   watchers: [
+    {
+      %{
+        table_name: "comments",
+        primary_key: "ID",
```

Comment:
Why is the `primary_key` column a binary while all other fields specifying columns use atoms?

Reply:
This is out-of-date 😅  I was allowing strings and atoms before.  I've just updated the README 👍